src/index.js

```diff
@@ -53,6 +55,16 @@ export function inject(container, options) {
     },
   });
   const workspace = Blockly.inject(container, options);
+  workspace.getRenderer().getConstants().selectedGlowFilterId = '';
+
+  const flyout = workspace.getFlyout();
+  if (flyout) {
+    flyout.getWorkspace().getRenderer().getConstants().selectedGlowFilterId = '';
+  }
+
+  const defs = container.querySelector('svg > defs');
```

Comment:
You don't have to actually use our defs. As long as you're referencing by ID the ref can exist in any SVG. personally if I were scratch I would probably want to keep my stuff separate. But I don't think it super matters.

Reply:
Should I just create a new empty SVG and tack these on to that?

Reply:
Yeah I think that makes sense!

Reply:
Added it to the workspace's root SVG.